erpnext/controllers/status_updater.py

```diff
@@ -36,6 +36,8 @@ def validate_status(status, options):
 	],
 	"Sales Order": [
 		["Draft", None],
+		["To Pick", "eval:self.per_picked > 1 and self.per_delivered < 100 and self.per_billed == 100 and self.docstatus == 1"],
```

Comment:
self.per_picked should be > 0 not 1. In case you have the per_picked as 0.5 this will fail.